scripts/samples/bench_and_plot.py

```diff
@@ -341,11 +341,11 @@ def bench(args: Namespace, bench_name: str, desc: str, generate: Callable,
     plt.grid(which='minor', ls=':', c='#ddd', lw=0.5)
     ax = plt.gca()
     ax.xaxis.set_major_formatter(ScalarFormatter())
-    ax.xaxis.get_major_formatter().set_scientific(False)
-    ax.xaxis.get_major_formatter().set_useOffset(False)
+    ax.xaxis.get_major_formatter().set_scientific(False)  # pyright: ignore
```

Comment:
Why do we need `# pyright: ignore` now since the you did not changed anything in this line ?

Reply:
Removed. Something to do about versions